spec/unit/actions/synchronize_etls_in_segment_spec.rb

```diff
@@ -7,21 +7,52 @@
 require 'gooddata/lcm/lcm2'
 require 'gooddata/lcm/actions/synchronize_etls_in_segment'
 
+def process_mock(name, id)
```

Comment:
warning: Similar code found in synchronize_etls_in_segment_spec.rb:10, synchronize_etls_in_segment_spec.rb:17 (mass = 42) (flay)